src/main/java/com/backend/dao/Database.java

```diff
@@ -0,0 +1,41 @@
+package com.backend.dao;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.model.movies.Movies;
+import com.utilities.FileReader;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class Database {
+
+    private List<Movies> moviesList = new ArrayList<>();
+
+    public List<Movies> getMoviesList() {
+        return moviesList;
+    }
+
+    public void setMoviesList(List<Movies> moviesList) {
+        this.moviesList = moviesList;
+    }
+
+    public void mapDataToMovies(String jsonFilePath) {
+
+        try {
+            byte[] jsonFile = FileReader.readJsonFile(jsonFilePath);
+
+            ObjectMapper mapper = new ObjectMapper();
+            Movies movies = mapper.readValue(jsonFile, Movies.class);
+
+            moviesList.add(movies);
```

Comment:
Do we only ever add 1 element to our `moviesList`? If so, why declare `moviesList` as a `List` at all?